macros/edr/system/system_utils/full_names.sql

```diff
@@ -62,7 +62,12 @@
 
 
 {% macro relation_to_full_name(relation) %}
-    {%- set full_table_name = relation.database | upper ~'.'~ relation.schema | upper ~'.'~ relation.identifier | upper %}
+    {%- if relation.database %}
+        {%- set full_table_name = relation.database | upper ~'.'~ relation.schema | upper ~'.'~ relation.identifier | upper %}
+    {%- else %}
+    {# Databricks doesn't always have a database #}
```

Comment:
There are cases when it does have database?
If no, then wouldn't it be better to use adapter here instead of if statement?

Reply:
There are cases when Databricks has a database, depending on user configuration.
Also wanted to be more platform agnostic thinking of future integrations.